catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/SearchService.java

```diff
@@ -216,8 +212,9 @@ public void executeQuery(Map<String, Object> queryMessage, Subject subject) {
         Set<String> sourceIds;
         if (StringUtils.equalsIgnoreCase(sources, LOCAL_SOURCE)) {
             LOGGER.debug("Received local query");
-            sourceIds = new HashSet<String>(Arrays.asList(searchController.getFramework()
-                    .getId()));
+            sourceIds =
+                    new HashSet<String>(Collections.singletonList(searchController.getFramework()
```

Comment:
If there is only one item in the collection, the guidance is to use `Collections.singletonList` instead.  Intellij will actually mark it as a warning.
https://stackoverflow.com/questions/26027396/arrays-aslist-vs-collections-singletonlist